go.mod

```diff
@@ -34,8 +33,8 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/prometheus/client_golang v0.9.2
 	github.com/prometheus/common v0.0.0-20181218105931-67670fe90761
-	github.com/prometheus/prometheus v0.0.0-20190118110214-3bd41cc92c78
-	github.com/prometheus/tsdb v0.4.0
+	github.com/prometheus/prometheus v0.0.0-20190328180107-4d60eb36dcbe
+	github.com/prometheus/tsdb v0.6.1
```

Comment:
Why this version not newest?

Reply:
I guess it's because 2.8.1 uses this? Then it's fine

Reply:
yes, you're right